Assets/Scripts/UI/DialogBox/Job List/DialogBoxJobList.cs

```diff
@@ -0,0 +1,53 @@
+#region License
+// ====================================================
+// Project Porcupine Copyright(C) 2016 Team Porcupine
+// This program comes with ABSOLUTELY NO WARRANTY; This is free software, 
+// and you are welcome to redistribute it under certain conditions; See 
+// file LICENSE, which is part of this source code package, for details.
+// ====================================================
+#endregion
+using System.Collections;
+using System.IO;
+using System.Linq;
+using System.Xml.Serialization;
+using UnityEngine;
+using UnityEngine.UI;
+
+////   Object -> MonoBehaviour -> DialogBox -> DialogBoxLoadSaveGame ->
+////                                                        DialogBoxSaveGame
+////                                                        DialogBoxLoadGame
+
+public class DialogBoxJobList : DialogBox
+{
+    public static readonly Color SecondaryColor = new Color(0.9f, 0.9f, 0.9f);
+
+    public GameObject jobListItemPrefab;
+    public Transform jobList;
+
+    public override void ShowDialog()
+    {
+        base.ShowDialog();
+        
+        foreach (Character c in World.Current.characters)
```

Comment:
@zwrawr Given the small amount of use of that var, I decided to keep it that way... As soon as the code involves more usage of that variable, I'll change it, indeed.